compiler/rustc_ast_pretty/src/pprust/state.rs

```diff
@@ -1138,23 +1129,29 @@ impl<'a> State<'a> {
                 let body = body.as_deref();
                 self.print_fn_full(sig, item.ident, gen, &item.vis, def, body, &item.attrs);
             }
-            ast::ItemKind::Mod(ref _mod) => {
+            ast::ItemKind::Mod(unsafety, ref mod_kind) => {
                 self.head(self.to_string(|s| {
                     s.print_visibility(&item.vis);
-                    s.print_unsafety(_mod.unsafety);
+                    s.print_unsafety(unsafety);
                     s.word("mod");
                 }));
                 self.print_ident(item.ident);
 
-                if _mod.inline || self.is_expanded {
-                    self.nbsp();
-                    self.bopen();
-                    self.print_mod(_mod, &item.attrs);
-                    self.bclose(item.span);
-                } else {
-                    self.s.word(";");
-                    self.end(); // end inner head-block
-                    self.end(); // end outer head-block
+                match mod_kind {
+                    ModKind::Loaded(items, ..) => {
```

Comment:
It looks like this will always pretty-print the contents of outline modules - is that intentional?

Reply:
For `--pretty=normal` and `--pretty=expanded` this is a replacement of the removed `self.is_expanded` flag.

For synthesizing tokens for inner macro attributes on modules (which are broken until https://github.com/rust-lang/rust/issues/81661 is fixed) I think it also makes sense to tokenize the whole module rather than just the `mod foo;` header.